app/components/documents/document-card.js

```diff
@@ -52,6 +52,8 @@ export default class DocumentsDocumentCardComponent extends Component {
   @tracked hasSignFlow = false;
   @tracked hasMarkedSignFlow = false;
 
+  @tracked label;
+
```

Comment:
there is already a getter called `label`.
```
  get label() {
    if (isPresent(this.args.label)){
      return this.intl.t(this.args.label);
    }
    return this.intl.t('uploaded-at');
  }
```
So unsure what ember is going to get, this new tracked value or the getter.
I would opt to call this tracked `altLabel` and only fill it when needed